README.md

```diff
@@ -99,9 +149,11 @@ open-attestation wrap ./examples/raw-documents/ ./examples/wrapped-documents/ --
 open-attestation wrap ./examples/raw-documents/ ./examples/wrapped-documents/ --oav3
 ```
 
-## Document privacy filter
+> **_NOTE:_** For transferable records, you should wrap them individually as each of them would be minted to a unique title escrow that represents the beneficiary and holder entities of the document. For more information about title escrow, refer [here](/docs/integrator-section/transferable-record/title-escrow).
```

Comment:
`(/docs/integrator-section/transferable-record/title-escrow)` dead link

Reply:
ah ok i fixed it. the new link will also be broken, until I update the documentation-website to the new structure.